con-base/src/main/java/ba/codecentric/base/service/ImageServiceImpl.java

```diff
@@ -0,0 +1,56 @@
+package ba.codecentric.base.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.UUID;
+
+@Service
+public class ImageServiceImpl implements ImageService {
+
+    @Value("${image.directory.path}")
+    private String directoryPath;
+
+    @Value("${image.timestamp.format}")
+    private String timestampFormat;
+
+    /**
+     * Saving an image from InputStream on the file system
+     * If the directory doesn't exist, it is created
+     * Name of the image is a timestamp and it is returned as a string
+     *
+     * @param image
+     * @param name
+     * @return
+     * @throws IOException
+     */
+    @Override
+    public String saveImage(InputStream image, String name) throws IOException {
+        Path path = Paths.get(directoryPath);
+        String newFileName;
+        try {
+            if (!Files.exists(path)) {
+                Files.createDirectories(path);
+            }
+            newFileName = getImageName(name);
+            Path filePath = Paths.get(directoryPath + newFileName);
+            Files.copy(image, filePath);
+        } finally {
+            image.close();
+        }
+
+        return newFileName;
+    }
+
+    public String getImageName(String oldName) {
```

Comment:
No need for `public` scope. You're only using it in `saveImage` method. Is there even a need for this method? Will it be reusable?